src/runtime_src/core/common/drv/include/xrt_cu.h

```diff
@@ -488,7 +488,9 @@ static inline void __xrt_cu_check(struct xrt_cu *xcu, bool force)
 	/* XRT CU assume command finished in order */
 	xcu->done_cnt += status.num_done;
 	xcu->ready_cnt += status.num_ready;
-	xcu->status = status.new_status;
+	// Do not update CU status if it has crashed
+	if (xcu->status != 0xffffffff)
```

Comment:
can we define a macro for 0xffffffff so that the code is readable and we can avoid comment?